### DISCUSSION TITLE
Allow player communication/ability to look around while unconscious

### DISCUSSION THREAD

**Discussion by FlyAwayNow** [acemod/ACE3#8510] [Q&A] [answered]
I was just wondering if we overlooked something and if it was possible that, when a player is in an unconscious state, we can keep them out of the black screen and looking around and allow them to keep on comms via ACRE, at least for casual gameplay

**Comment by jonpas** [comment, marked as answer]
Negative, cursor is locked. The only way you can look around is with headtracking, as we can't really block that.
